doc/build.rst

```diff
@@ -130,10 +135,13 @@ Building the Targets
    * ``SIMULATE``: Set this parameter to ``ON`` to build the enclave in simulation mode (for local development and testing, in case your machine does not support hardware enclaves). This parameter requires ``OE_DEBUG`` to be set to 1.
    * ``OBLIVIOUS``: Set this parameter to ``ON`` to perform model training and inference using data-oblivious algorithms (to mitigate access-pattern based side-channel attacks).
 
-   Finally, we also provide options to build the library with LVI mitigation. To enable LVI mitigation, set the option ``LVI_MITIGATION`` to ``ON``, and set the variable ``LVI_MITIGATION_BINDIR`` to point to the location where you installed the LVI mitigated Open Enclave libraries.
+   Finally, we also provide options to build the library with LVI mitigation.
+   
+   * ``LVI_MITIGATION``: Set this to ``ON`` to enable LVI mitigation. 
+   * ``LVI_MITIGATION_BINDIR``: Set this variable to point to the location where you installed the LVI mitigated Open Enclave libraries.
 
 
-3. On Ubuntu, build the Secure XGBoost targets by running CMake:
+3. **Build the Secure XGBoost targets** by running CMake:
```

Comment:
Same thing here. Perhaps we can drop the "by running CMake", since the code block below should make that evident.